sql/moz-fx-data-shared-prod/firefox_ios_derived/app_store_retention_week_2_v1/query.sql

```diff
@@ -0,0 +1,48 @@
+WITH clients_retention AS (
+  SELECT
+    submission_date,
+    client_id,
+    sample_id,
+    days_seen_bits,
+  FROM
+    firefox_ios.baseline_clients_last_seen
+  WHERE
+    submission_date = @submission_date
+    AND normalized_channel = "release"
+),
+first_seen AS (
+  SELECT
+    client_id,
+    sample_id,
+    first_seen_date,
+  FROM
+    firefox_ios.baseline_clients_first_seen
```

Comment:
@kik-kik and @soGaussian we spent a bunch of time in the last sprints working on the `firefox_ios_clients` and `firefox_android_clients` tables and figuring out how we can calculate new profiles from it. We aligned on the new profile calculation and did work to validate if it matches up to `active_users_aggregates` as the way we calculate new profiles. From this it looks like we are choosing not to use the `ios_clients` table. Is there a reason why it won't fit our needs to calculate the cohort? Think through some of the effects this might end up having e.g. what if the cohort calc here starts to deviate from the new profiles calc? ios clients has attribution - do we want to implement attribution in two different places?

Reply:
So the main thing I'd be careful about when using `ios_clients` table as the base table is this:

https://github.com/mozilla/bigquery-etl/blob/main/sql/moz-fx-data-shared-prod/firefox_ios_derived/firefox_ios_clients_v1/query.sql#L167-L194

The problem is that all those attributes we set for a client in the clients table is done over time giving us a 2 week window before this query would run against a user for the first time. I would assume this should not be a problem at this time frame, but lets say we'd want to add another attribution metric for week 1 or even day 1 it would potentially mean that at the time when this query would run some of the user attributes could be missing.

It's also not the end of the world since we could assume that the later funnel calculations are more up-to-date and complete (lets say the week_2 retention table is not expected to see this behaviour). This is something we could resolve in the view combining all the data.